src/frontend/komponenter/Fagsak/Vilkaar/velgvilkaarene.less

```diff
@@ -0,0 +1,6 @@
+.vilkaarene {
+    &__navigering {
```

Comment:
This could potentially be a part of our parent navigation komponent.

Reply:
yes, agree